canvas_modules/common-canvas/src/common-properties/components/editor-form/editor-form.jsx

```diff
@@ -51,6 +51,7 @@ class EditorForm extends React.Component {
 		this.state = {
 			showFieldPicker: false
 		};
+		this.showAlertsTab = props.controller ? get(props.controller.getPropertiesConfig(), "showAlertsTab", true) : true;
```

Comment:
Why not pass this down from properties-main instead?  Similar to what we do with 
```
rightFlyout={this.props.propertiesConfig.rightFlyout}
```